<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>273</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Calculdora IMC</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Estatura</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textEstatura">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Peso</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textPeso">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxSeleccionar">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>230</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>cm y Kg</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>pul y libras</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btnCalcular">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Calcular</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLimpiar">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>290</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Limpiar</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="resultado">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>251</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnLimpiar</sender>
   <signal>clicked()</signal>
   <receiver>textPeso</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnLimpiar</sender>
   <signal>clicked()</signal>
   <receiver>textEstatura</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>221</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnLimpiar</sender>
   <signal>clicked()</signal>
   <receiver>resultado</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>152</x>
     <y>406</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
